src/server/pps/server/api_server.go

```diff
@@ -652,6 +652,12 @@ func (a *apiServer) ListJobSet(request *pps.ListJobSetRequest, serv pps.API_List
 	// Track the jobsets we've already processed
 	seen := map[string]struct{}{}
 
+	// Create a filter function based on the request
```

Comment:
This comment (and the similar one at l. 831) may not add a lot (a bit like `x = 2 * x // double x`), but neither is wrong.